src/clear/sql/connection_pool.cr

```diff
@@ -7,18 +7,24 @@ class Clear::SQL::ConnectionPool
     @@databases[name] = DB.open(uri)
   end
 
+  # Specify whether to refresh the connection upon error
+  class_property? reconnect = ENV["PG_RECONNECT"]?.presence.try(&.downcase).in?("1", "true")
```

Comment:
```suggestion
  class_property? reconnect : Bool = ENV["PG_RECONNECT"]?.presence.try(&.downcase).in?("1", "true")
```

Reply:
cannot infer type here

Reply:
I don't think I ended up using the property